pandas/tests/test_algos.py

```diff
@@ -648,7 +648,9 @@ def test_value_counts_uint64(self):
         expected = Series([1, 1], index=[-1, 2**63])
         result = algos.value_counts(arr)
 
-        tm.assert_series_equal(result, expected)
+        # 32-bit linux has a different ordering
```

Comment:
This is indicative of a flakiness, though I'm not sure how to secure proper ordering in this case.